ui/src/app/machines/views/MachineList/StatusColumn/StatusColumn.js

```diff
@@ -99,17 +100,63 @@ const getStatusIcon = machine => {
 };
 
 const StatusColumn = ({ onToggleMenu, systemId }) => {
+  const dispatch = useDispatch();
   const machine = useSelector(state =>
     machineSelectors.getBySystemId(state, systemId)
   );
   const osReleases = useSelector(state =>
     generalSelectors.osInfo.getOsReleases(state, machine.osystem)
   );
+  let actionLinks = [];
+  const actionTypes = new Map([
+    ["abort", null],
+    ["acquire", null],
+    ["commission", null],
+    ["deploy", null],
+    ["exit-rescue-mode", "exitRescueMode"],
+    ["lock", null],
+    ["mark-broken", "markBroken"],
+    ["mark-fixed", "markFixed"],
+    ["override-failed-testing", "overrideFailedTesting"],
+    ["release", null],
+    ["rescue-mode", "rescueMode"],
+    ["test", null],
+    ["unlock", null]
+  ]);
+  Array.from(actionTypes.keys()).forEach(action => {
+    if (machine.actions.includes(action)) {
+      actionLinks.push({
+        children: `${action}...`,
```

Comment:
In `general.machineActions.data` each action has a display name unhelpfully called `title`. Can we use that here instead of the raw action name?

Reply:
That's a good idea, I will do that as a follow up as part of https://github.com/canonical-web-and-design/maas-ui/issues/793.